.github/workflows/check.yaml

```diff
@@ -0,0 +1,37 @@
+name: Run static analysis
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  run-static-analysis:
+    if: |
+      github.event_name == 'push' ||
+      github.event_name == 'pull_request' &&
+      github.event.pull_request.head.repo.full_name != github.repository
+
+    runs-on: ubuntu-22.04
+
+    steps:
+    - uses: actions/checkout@master
+
+    - name: Setup Tarantool CE
+      uses: tarantool/setup-tarantool@v3
+      with:
+        tarantool-version: '3.2.0'
+
+    - name: Setup tt
+      run: |
+        curl -L https://tarantool.io/release/3/installer.sh | sudo bash
+        sudo apt install -y tt
+        tt version
+
+    - name: Setup luacheck
+      run: |
+        tt rocks install luacheck 0.26.0
+
+    - run: echo $PWD/.rocks/bin >> $GITHUB_PATH
+
+    - name: Run luacheck
```

Comment:
I think we should run full `make check` here, since we removed `make check` from the other workflow.

Reply:
Ok, but `check` is an alias for `luacheck`:

https://github.com/tarantool/metrics-export-role/blob/d31d99a4e5d84b9de9d386374dface43bc75ae1e/Makefile#L33-L36

Reply:
For now it is, but it is possible that later it would be something like
```
check: luacheck codespell
```

Reply:
Yes, corrected.